src/extension.ts

```diff
@@ -36,6 +36,8 @@ import { LazyProgress } from "./lazy-progress";
 import * as fs from "fs";
 
 export async function activate(context: ExtensionContext) {
+  detectLaunchConfigurationChanges();
```

Comment:
Note: this is intentionally invoked early on.

If the launch goes wrong it's likely related to configuration, so you will be fiddling with those values.

Reply:
👍